wordcloud/wordcloud.py

```diff
@@ -503,16 +508,17 @@ def process_text(self, text):
 
         flags = (re.UNICODE if sys.version < '3' and type(text) is unicode
                  else 0)
-        regexp = self.regexp if self.regexp is not None else r"\w[\w']+"
+        regexp = self.regexp if self.regexp is not None else r"(\w[\w']+|\w+)"
```

Comment:
Sorry for be late, I was completing my career studies and then on vacations. Let's make it clear, and list which cases should cover the regex in order to know how to build it (you should say which ones are desirable):

**ACCEPT**
- [ ] words with more than 1 char, possibly containing numbers (e.g. "word", "w0rd").
- [ ] words having the character ' literally (e.g. "O'Reilly", "didn't")
- [ ] numbers (e.g. "1", "2", "2017")

**REFUSE**
- [ ] words with a single char which is not a number (e.g. "A", "i", "x")

That makes sense? Once we know the expected behavior of the regex, then we can build it and test all the code with those changes.